src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java

```diff
@@ -0,0 +1,285 @@
+package edu.caltech.ipac.firefly.server.dpanalyze;
```

Comment:
Sorry, I missed this in @robyww PR.  As a reminder, he should create new Java packages directly under firefly and not server.  Everything in firefly is 'server' now.

Also, consider where best to put all future analyzers.  If it's placed here, then it may clutter the core classes and interface, making it harder to distinguish core vs impl.
I would think it belongs with the other sofia classes.

Reply:
I did not know that was out strategy on the server side. It is fine but we should talk about it.